serverless-multi-tenant-api/README.md

```diff
@@ -0,0 +1,88 @@
+# Multi-Tenant Serverless API
+
+This project demonstrates a multi-tenant serverless API using API Gateway, Lambda, and DynamoDB. Requests are authorized using API keys mapped to tenants in DynamoDB. Access to data is restricted using dynamic IAM policies based on the tenant ID. 
+
+## Architecture
+
+The project contains the following resources:
+
+- API Gateway REST API
+- Lambda Authorizer Function - Looks up the tenant ID from the API key and generates IAM policies  
+- Lambda GET Function - Retrieves items by ID after validating the tenant ID
+- Lambda POST Function - Adds new items after validating the tenant ID
+- DynamoDB Tables - Stores tenant data isolated by partition keys with the tenant ID
+
+The Lambda functions assume IAM roles with temporary credentials to access the tenant's data partition.
+
+## Setup
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+1. Change directory to the pattern directory:
+    ```
+    cd serverless-patterns/serverless-multi-tenant-api
+    ```
+1. From the command line, use AWS SAM to deploy the AWS resources for the workflow as specified in the template.yaml file:
+
+    ```
+    npm install
+    sam build
+    sam deploy --guided
+    ```
+
+This will build and deploy the Lambda functions, API Gateway API, and DynamoDB tables.
+The API endpoint will be output after deployment. You can test the endpoint with valid API keys and tenant IDs stored in DynamoDB.
+
+Set up a tenant in the ConfigTable DynamoDB Table with the generated API Key:
```

Comment:
```suggestion
Set up a tenant in the ConfigTable DynamoDB table with the generated API Key:
```

Reply:
Please add a little more context so that users who might not be that familiar with DynamoDB know how/where to enter this